RomM/platform_maps.py

```diff
@@ -192,3 +195,24 @@
 SPRUCEOS_SUPPORTED_PLATFORMS_FS = frozenset(
     SPRUCEOS_SUPPORTED_PLATFORMS_FS_MAP.values()
 )
+
+_env_maps = None
+_env_platforms = None
+
+def _load_env_maps() -> dict[str, str]:
+    raw = os.getenv("CUSTOM_MAPS")
+    if not raw:
+        return {}
+    try:
+        loaded = json.loads(raw)
+        return loaded
+    except json.JSONDecodeError as e:
+        raise ValueError(f"Invalid JSON in CUSTOM_MAPS: {e}")
```

Comment:
IYO should this raise an error and crash or print to the log file and return `{}`?

Reply:
I forgot to go back and resolve this. I put one except for json parsing errors and another for unexpected errors. In both cases, I have returned {}. 

On a somewhat related tangent, I tried to think of a way to report unmapped directories to help nudge you in the right direction for this configuration. For instance, if you had Playstation as "psx" in your file system but the slug is "ps" I thought it might be useful to say "hey man, you've got stuff in RomM at 'ps' that doesn't map to a directory in your file system. Consider using CUSTOM_MAPS to resolve this discrepancy" but I think that is outside of the scope of this PR